test/test_lzma.py

```diff
@@ -8,8 +8,12 @@
     # Try the location used on Python 3 first,
     from test.support import _4G, TESTFN, import_module, bigmemtest, run_unittest, unlink
 except ImportError:
-    # Must be under Python 2 then,
-    from test.test_support import _4G, TESTFN, import_module, bigmemtest, run_unittest, unlink
+    try:
+        # May be < Python 2.7.14
+        from test.test_support import _4G, TESTFN, import_module, bigmemtest, run_unittest, unlink
+    except ImportError:
+        # May be >=Python 2.7.14, <3
+        from future.backports.test.support import _4G, TESTFN, import_module, bigmemtest, run_unittest, unlink
```

Comment:
With Python 2.7.14, this fallback should be unnecessary. Though `defaults` will need to rebuild with a patch. Just wrote it for `conda-forge`. Please see PR ( https://github.com/conda-forge/python-feedstock/pull/168 ) for details.

Reply:
Well, for some reason the python 2.7.14 in Fedora 27 also doesn't have this, neither does the (library) docker container for python 2.7.14. The only distro where I found this working was in ArchLinux.

Reply:
I reported it here: https://bugzilla.redhat.com/show_bug.cgi?id=1528899

Reply:
Yeah, I think a lot of people were caught by surprise. It's not really a breaking change, but still it requires some changes for those of us aggressively cleaning `test`. Thanks for raising the issue.

Reply:
Should add that the fallback is harmless (and helpful in some cases). So it is reasonable to keep.